# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/ipc/NeckoParent.cpp
# Commit: 2081d42e6ccd
# Full Hash: 2081d42e6ccdcc39b3d1b2e6523a8d44fed85390
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2024-09-13 04:28:43
# Regressor Bug: 1475599
# File Overlap Count: 1
# Description:
#   Bug 1475599 - part 8 - CookieStore API - e10s support, r=smaug,cookie-reviewers,timhuang
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D217574
# ==============================================================================

diff -r c6e20b939a21 -r 2081d42e6ccd netwerk/ipc/NeckoParent.cpp
--- a/netwerk/ipc/NeckoParent.cpp	Thu Sep 12 16:48:21 2024 +0000
+++ b/netwerk/ipc/NeckoParent.cpp	Thu Sep 12 16:48:21 2024 +0000
@@ -287,7 +287,7 @@
 }
 
 PCookieServiceParent* NeckoParent::AllocPCookieServiceParent() {
-  return new CookieServiceParent();
+  return new CookieServiceParent(static_cast<ContentParent*>(Manager()));
 }
 
 bool NeckoParent::DeallocPCookieServiceParent(PCookieServiceParent* cs) {
